components/brave_rewards/browser/test/rewards_browsertest.cc

```diff
@@ -112,6 +112,7 @@ class RewardsBrowserTest : public InProcessBrowserTest {
 };
 
 IN_PROC_BROWSER_TEST_F(RewardsBrowserTest, RenderWelcome) {
+  base::ScopedAllowBlockingForTesting allow_blocking;
```

Comment:
how come this is needed now? This was not a problem before and nothing was added to rewards flow. We need to find another way for this one as this now needs to be added to every test